ga4gh/datamodel/variants.py

```diff
@@ -269,6 +269,7 @@ class HtslibVariantSet(datamodel.PysamDatamodelMixin, AbstractVariantSet):
     """
     def __init__(self, id_, dataDir):
         super(HtslibVariantSet, self).__init__(id_)
+        self._dataDir = dataDir
```

Comment:
I take it back.  `DatamodelObject`s aren't necessarily backed by directories because it can have simulated subclasses.  (I suppose we could initialize the dataDir with None in the case of simulated, etc. subclasses that didn't need a dataDir, but that seems ugly.)  The mixin does seem like the logical place to put it, but again, I don't know if we want to "initialize" a mixin.